indra/newview/llavatarrendernotifier.h

```diff
@@ -33,6 +33,16 @@
 
 class LLViewerRegion;
 
+enum ERezzedStatus
```

Comment:
this was modified in another branch (maint a?) and is going to cause conflicts.

Reply:
Also feel like a wrong place for this particular enum. It's avatar status, notifier should use it, not declare it.

Reply:
> Also feel like a wrong place for this particular enum

The file llavatarrendernotifier.h is imcluded in llavatar.h
Thus the backward reference doesn't work
Forward reference works on win but fails on mac

Reply:
> this was modified in another branch (maint a?) and is going to cause conflicts.

This PR is targeted to [DRTVWR-600-maint-A](https://github.com/secondlife/viewer/tree/DRTVWR-600-maint-A)

Reply:
Sorry, I was under impression that it is for maint C. Maint A is effectively closed for non-urgent fixes. Since previous maint released, DRTVWR-600-maint-A it is about to go as RC.

Reply:
> Forward reference works on win but fails on mac

You likely need to declare it as enum ERezzedStatus : S32 for forward declaration to work. Forward declaration of enums is withing C++11 standard, but it might need to have specified size on both sides.

P.S. I really like the idea of having it as an enum.